src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "../../address-registries/interfaces.sol";
+import "../../sequencer/SetSequencerInboxMaxTimeVariationAction.sol";
+
+contract AIPSetSequencerInboxMaxTimeVariationArbOneAction is
+    SetSequencerInboxMaxTimeVariationAction
+{
+    // TODO: confirm / finalize values
+    constructor()
+        SetSequencerInboxMaxTimeVariationAction(
+            ISequencerInboxGetter(0xd514C2b3aaBDBfa10800B9C96dc1eB25427520A0),
```

Comment:
Why don't we use the address registry like in other action contracts?

Reply:
this is the address registry — it's instantiated w/ the interface w/ only the getter we need